### PR TITLE
Fix/map 91/refactoring routes after login

### PR DESCRIPTION
**Pull request by mazursasha1990**

### CODE REVIEW

File: backend/libs/strategies/facebookStrategy.ts [hunk 1]
```diff
@@ -15,8 +15,8 @@ const facebookStrategy = new FacebookStrategy(
       const email = profile?.emails?.[0].value;
       let user = null;
 
-      if (email) {
-        user = await UserModel.findOne({ email });
+//       if (email) {
+//         user = await UserModel.findOne({ email });
```
Comment (steadyGuy) [inline backend/libs/strategies/facebookStrategy.ts]: ??

File: backend/libs/strategies/facebookStrategy.ts [hunk 2]
```diff
@@ -40,4 +40,4 @@ const facebookStrategy = new FacebookStrategy(
   }
 );
 
-export default facebookStrategy;
+// export default facebookStrategy;
```
Comment (steadyGuy) [inline backend/libs/strategies/facebookStrategy.ts]: ?

File: frontend/src/redux/reducers/index.tsx [hunk 3]
```diff
@@ -3,12 +3,14 @@ import { userReducer } from './userReducer';
 import { userLoginReducer } from './userAuthReducer';
 import { popupLocationReducer } from './popupLocationReducer';
 import { locationsListReducer } from './locationListReducer';
+// import { filterReducer } from './filtersReducer';
```
Comment (steadyGuy) [inline frontend/src/redux/reducers/index.tsx]: I don't really understand, why filters is commented

File: frontend/src/redux/reducers/index.tsx [hunk 4]
```diff
@@ -3,12 +3,14 @@ import { userReducer } from './userReducer';
...
 
 export const rootReducer = combineReducers({
   user: userReducer,
   userAuth: userLoginReducer,
   popupLocation: popupLocationReducer,
   locationList: locationsListReducer
+  // userFilters: filterReducer
```
Comment (steadyGuy) [inline frontend/src/redux/reducers/index.tsx]: filters commented

**Review by steadyGuy — APPROVED** [review]

**Comment by steadyGuy** [comment]
Still, we have flickering, but it's not very important

**Review by 6mahdapihka9 — APPROVED** [review]

**Comment by 6mahdapihka9** [comment]
change in next sprint mapUserProps function on backend, so it will match data type on frontend

**Pull request merged**